topics/metabolomics/tutorials/gcms/tutorial.md

```diff
@@ -0,0 +1,432 @@
+> >  - **Method to use for grouping** : please `keep the PeakDensity method` which will groups peak based on time dimension peak densities.
+> >  - **Bandwidth** : it correspond to the standard deviation of the smoothing kernel to be used. Set it to `10` for our example.
+> >  - **Minimum fraction of samples** : minimum fraction of samples in at least one sample group in which the peaks have to be present to be considered as a peak group. This value is the threshold and `don't change` it for our example.
+> >  - **Minimum number of samples** : minimum number of sample(s) in which the peak have to be detected to be considered as a group. Please `don't change` it for our example.
+> >  - **Width of overlapping m/z slices** : the m/z difference between two peaks to be able to be grouped. For our example please set it to `0.05`.
+> > 
+> > You don't need to change the other parameters, except if you want the peaklist as output. You should now have these files in your history : 
+> >  - **Input** : 
+> >    - the RData file obtained just before with all datas and peaks informations for each files (format : `xset.merged.RData`).
+> >  - **Output** : 
+> >    - one Rdata file named `xset.merged.groupChromPeaks.RData` containing all peaks grouped according to their mz and retention time.
+> >    - one `pdf` file named `xset.merged.groupChromPeaks.plotChromPeakDensity.pdf`. It contains chromatographic density plots.
+> >    - 2 `tsv` files. The first one named `xset.merged.group.dataMatrix.tsv` contains informations about ions intensities. The second one named `xset.merged.group.variableMetadata.tsv` contains an other table with informations about ions. These files are generated **only if you set the parameter *Get peaklist*** to `yes`. It is just more informations after the grouping step. 
+> {: .solution}
+>
+> > ### {% icon solution %} 7 - (Optionnal) XCMS step : *integrating areas of missing peaks*
+> >
+> > This last step can be run after grouping your peaks and don't need the retention time correction. It is not an obligation to process it but when you have low peaks it can be helpfull to obtain something around them. Run **xcms fillChromPeaks (fillPeaks)** {% icon tool%} to identify, for each sample, peak groups where that sample is not represented. 
+> >  - **Input** : 
+> >    - your Rdata file `xset.merged.groupChromPeaks.*.RData`
+> >  - **Output** : 
+> >    - a new RData file named `xset.merged.groupChromPeaks.*.fillChromPeaks.RData`
+> {: .solution}
+> > ### {% icon comment %} Important : Be careful of the file format
+> >
+> > During each step of pre-processing, your file has its format changed and can have also its name changed.
+> > To be able to continue to MSMS processing, you need to have a RData object wich is **merged and grouped** (step 4 and 5) at least. It means that you should have a file named `xset.merged.groupChromPeaks.RData` (and maybe with some step more in it).
+> {: .warning} 
+{: .details}
+
```

Comment:
Since this section can be considered as undertaken in the  [Mass spectrometry: LC-MS preprocessing with XCMS training](https://training.galaxyproject.org/training-material/topics/metabolomics/tutorials/lcms-preprocessing/tutorial.html), I suggest recommending the users to start with the previous training and skip this part; in my opinion, it would be better to provide the required datasets (generated in the previous training) as Zenodo links, and start the training by the **Verify your data after the XCMS pre-processing**.

Reply:
Hi @gallardoalba , Many many thanks for the work. You're right it will be simpler this way.

Reply:
Thank you a lot @gallardoalba for this very complete review!
Concerning this specific comment, indeed it may be partly redundant with the "Mass spectrometry: LC-MS preprocessing with XCMS" training,, and thus simplification is a good suggestion. 
I checked in more details what would be needed to replace this part, and it appears that simply providing the required data may not be satisfactory: in fact the dataset used in the two tutorials are different and due to the way the specific RData file used are handled in modules, it would imply to provide anyway the starting files from the XCMS file anyway. 
Thus, I would suggest to keep a "hands-on" about the required XCMS steps but in a "lighter" format. By this, I mean inviting the trainee to follow the first step of the LC-MS tutorial to get familiar with the steps and parameters, and then providing a hands-on to run the needed steps on the GC-MS dataset but only containing the parameters to run with no additional explanation, considering the explanations are given in the LC-MS training. 
Would this seems good?
If it is alright, I can propose something for this part, if it is also ok for you @yguitton, I have some ideas.

Reply:
@melpetera Ok for me

Reply:
Also for me :)

Reply:
Done in 7da7eca :)
I chose not to put the hands-on as only one block to help trainees in grasping the main steps. Thus I put the "1- / 2- / 3-" main parts as sections / sub-sections to make it simplier. 
The jobs are the same without the redundant explainations. 
This ok?

Reply:
Looks fine for me.